### PR TITLE
Add Mellanox SN2410.yml

### PR DESCRIPTION
**Pull request by mihacucek**

### CODE REVIEW

**Comment by mihacucek** [comment]
I have updated the code with indentations that were missing...

**Comment by mihacucek** [comment]
removed space ad the end reported in error

Error: types/Mellanox/SN2410.yml:323:20: [error] trailing spaces (trailing-spaces)

**Review by DanSheps — APPROVED** [review]

File: device-types/Mellanox/SN2410.yml [hunk 1]
```diff
@@ -0,0 +1,323 @@
+---
+manufacturer: Mellanox
+model: SN2410
+slug: sn2410
+part_number: ''
```
Comment (DanSheps) [inline device-types/Mellanox/SN2410.yml:5]: Fill this in please

File: device-types/Mellanox/SN2410.yml [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+u_height: 1
+is_full_depth: true
+subdevice_role: ''
```
Comment (DanSheps) [inline device-types/Mellanox/SN2410.yml:8]: This needs to be removed or set as "parent"

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @mihacucek

**Comment by mihacucek** [comment]
Hi Daniel,
The question is if project needs contributions like this :-).

I tried to contribute to the project by adding device type but got error messages all the time, regardles of correcting the mistakes.
After a few tries I stopped.

Feel free to delete or close this.

BR,
Miha

**Comment by danner26** [comment]
If you still want to contribute we would be happy to review it again. The original maintainers are busy but I am here to make sure the PRs get worked on
If you would like to contribute please feel free to reopen this PR, update your branch against our master, and then fix any issues. From there I will review it. If you run into issues you can also give me access to your repo, and I will be happy to correct any issues that are preventing the CI from running.

**Pull request closed without merging**